blockly-ws-search/src/css.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Styling for workspace search.
+ * @author aschmiedt@google.com (Abby Schmiedt)
+ * @author kozbial@google.com (Monica Kozbial)
+ */
+
+/**
+ * Base64 encoded data uri for close icon.
+ * @type {string}
+ */
+const CLOSE_SVG_DATAURI =
+    'data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC' +
+    '9zdmciIGhlaWdodD0iMTQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjE0Ij48cGF0aC' +
+    'BkPSJNMTkgNi40MUwxNy41OSA1IDEyIDEwLjU5IDYuNDEgNSA1IDYuNDEgMTAuNTkgMTIgNS' +
+    'AxNy41OSA2LjQxIDE5IDEyIDEzLjQxIDE3LjU5IDE5IDE5IDE3LjU5IDEzLjQxIDEyeiIvPj' +
+    'xwYXRoIGQ9Ik0wIDBoMjR2MjRIMHoiIGZpbGw9Im5vbmUiLz48L3N2Zz4=';
+
+/**
+ * Base64 encoded data uri for keyboard arrow down icon.
+ * @type {string}
+ */
+const ARROW_DOWN_SVG_DATAURI =
+    'data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC' +
+    '9zdmciIGhlaWdodD0iMTQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjE0Ij48cGF0aC' +
+    'BkPSJNNy40MSA4LjU5TDEyIDEzLjE3bDQuNTktNC41OEwxOCAxMGwtNiA2LTYtNiAxLjQxLT' +
+    'EuNDF6Ii8+PHBhdGggZD0iTTAgMGgyNHYyNEgwVjB6IiBmaWxsPSJub25lIi8+PC9zdmc+';
+
+/**
+ * Base64 encoded data uri for keyboard arrow up icon.
+ * @type {string}
+ */
+const ARROW_UP_ARROW_SVG_DATAURI =
+    'data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC' +
+    '9zdmciIGhlaWdodD0iMTQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjE0Ij48cGF0aC' +
+    'BkPSJNNy40MSAxNS40MUwxMiAxMC44M2w0LjU5IDQuNThMMTggMTRsLTYtNi02IDZ6Ii8+PH' +
+    'BhdGggZD0iTTAgMGgyNHYyNEgweiIgZmlsbD0ibm9uZSIvPjwvc3ZnPg==';
+
+/**
+ * CSS for workspace search.
+ * @type {Array.<string>}
+ */
+const CSS_CONTENT = [
+  'path.blocklyPath.search-highlight {',
+    'fill: black;',
+  '}',
+  'path.blocklyPath.search-highlight.search-current {',
+    'fill: grey;',
+  '}',
+  '.blockly-btn-text {',
+    'height: 1px;',
+    'overflow: hidden;',
+    'position: absolute;',
+    'width: 1px;',
+  '}',
+  '.blockly-close-btn {',
```

Comment:
Why is it that some are prefixed with ws-search, and others are not?

Reply:
Yeah looking back I was trying to follow some good conventions for css naming and then just didn't apply that to the buttons. Planning on adding blockly-ws-search as prefix to the buttons. Sound ok?